source/tutorial/swiftui.txt

```diff
@@ -351,31 +362,34 @@ Change the Subscription
       Return to your mobile app. Because we added ``reRunOnOpen``, the app
       should re-sync only the documents that match the Flexible Sync query.
       After an initial moment when Realm resyncs the document collection, you 
-      will see the new Item of High priority that you created.
+      will see only the new Item of High priority that you created.
+
+      You'll note that the document you initially created is not synced, 
+      because it does not have a ``priority`` field. If you want this Item to 
+      be synced, you can edit the document in the Atlas UI and add a value for 
+      the priority field.
       
       If you want to further test the functionality, you can create Items of various 
       priorities. You will see that a new Item with a lower priority briefly
-      appears in the list of Items and then disappears. You can see a message
-      similar to this one in the logs:
+      appears in the list of Items and then disappears. The Sync error handler
+      helpfully provides a message describing this behavior:
 
       .. code-block:: sh
          :copyable: false
 
-         2022-08-08 18:00:45.726138-0400 App[18133:242396] Sync: Connection[2]: 
-         Session[2]: Received: ERROR "Client attempted a write that is outside 
-         of permissions or query filters; it has been reverted" 
-         (error_code=231, try_again=true, recovery_disabled=false)
+         ERROR 
+         "Client attempted a write that is outside 
+         of permissions or query filters; it has been reverted"
 
-      Realm creates the Item locally, syncs it with the backend, and then reverts 
-      the write because it doesn't meet the subscription rules.
+      You can also see this message in the console log.
 
-      You'll note, too, that the document you initially created is not synced, 
-      because it does not have a ``priority`` field. If you want this Item to 
-      be synced, you can edit the document in the Atlas UI and add a value for 
-      the priority field.
+      In this instance, Realm creates the Item locally, syncs it with the 
```

Comment:
```suggestion
      In this situation, Realm creates the Item locally, syncs it with the 
```

nit: had to read 2x b/c when i read 'instance', i thought 'instance of object/class'. not super important. also not in love w word 'situation' as in the suggestion, if you have alternatives.

Reply:
That's fair - went with scenario.